roles/compliance/tasks/install.yml

```diff
@@ -0,0 +1,142 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Install the compliance operator
+- name: Create the compliance namespace
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit) }}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    api_version: v1
+    kind: Namespace
+    state: present
+    definition:
+      kind: Namespace
+      metadata:
+        name: "{{ cifmw_compliance_namespace }}"
+        labels:
+          openshift.io/cluster-monitoring: "true"
+          pod-security.kubernetes.io/enforce: privileged
+  when: not cifmw_compliance_dry_run | bool
+
+- name: Ensure OperatorGroup for the compliance-operator is present
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit) }}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    state: present
+    wait: true
+    definition:
+      apiVersion: operators.coreos.com/v1
+      kind: OperatorGroup
+      metadata:
+        name: compliance-operator
+        namespace: "{{ cifmw_compliance_namespace }}"
+      spec:
+        targetNamespaces:
+          - "{{ cifmw_compliance_namespace }}"
+  when: not cifmw_compliance_dry_run | bool
+
+- name: Ensure Subscription for the compliance-operator is present
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit)}}"
+    state: present
+    wait: true
```

Comment:
as above

Reply:
yeah - I copied this from the test operator -- so that should be fixed there too. I haven't found the appropriate conditions to wait for, but I don't think I need to wait here anyways. The real wait can happen later on when we use the csv. So , removing wait here.